functions/src/import-employees-from-alibeez.ts

```diff
@@ -47,11 +48,9 @@ export const importEmployeesFromAlibeez = async (config: AlibeezConfig) => {
     return;
   }
   const employees: AlibeezEmployee[] = await response.json();
-  const employeesWithValidEmail = employees.filter(
-    employee =>
-      employee.zenikaEmail && employee.zenikaEmail.endsWith("@zenika.com")
-  );
-  const importRef = firebase
+  const[ employeesWithValidEmail, employeesWithNoValidEmail ] = partition(employees,function (o) { return o.zenikaEmail && o.zenikaEmail.endsWith('@zenika.com') })
+  console.info('Employees with no valid email : '+ employeesWithNoValidEmail)  
```

Comment:
I'm wondering if logging one line per email would be even better. Let's discuss it.